src/cljs/memory_hole/widgets/tag_editor.cljs

```diff
@@ -25,7 +25,7 @@
 (defn- filter-selections [selections all-tags issue-tags user-input]
   (->> (difference all-tags issue-tags)
        (sort)
-       (filter #(string/includes? (string/lower %) (some-> user-input string/lower)))
+       (filter #(string/includes? (string/lower-case %) (some-> user-input string/lower-case)))
```

Comment:
this needs to do nil checks if you switch to clojure.string

Reply:
ok, got it